### ISSUE TITLE
False positive for "Couldn't find the file readme.md"

### ISSUE THREAD

**Issue by carlosperate** [sindresorhus/awesome-lint#33]
Currently getting this error with awesome-lint@0.3.0, when the `README.md` is definitely there:

```
$ awesome-lint
Error: Couldn't find the file readme.md
    at lint (/home/travis/.nvm/versions/node/v10.8.0/lib/node_modules/awesome-lint/index.js:20:25)
    at Function.lint.report (/home/travis/.nvm/versions/node/v10.8.0/lib/node_modules/awesome-lint/index.js:30:21)
    at main (/home/travis/.nvm/versions/node/v10.8.0/lib/node_modules/awesome-lint/cli.js:42:20)
    at Object.<anonymous> (/home/travis/.nvm/versions/node/v10.8.0/lib/node_modules/awesome-lint/cli.js:49:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
```

Travis file: https://github.com/carlosperate/awesome-microbit/blob/fccbad1b4b2bf503f2284e097eedb092be8c0751/.travis.yml
Travis output: https://travis-ci.org/carlosperate/awesome-microbit/builds/414668054
Readme: https://github.com/carlosperate/awesome-microbit/blob/master/README.md
Repo: https://github.com/carlosperate/awesome-microbit

**Comment by transitive-bullshit** [comment]
`awesome-lint https://github.com/carlosperate/awesome-microbit` works fine.
`awesome-lint README.md` also works fine because you're telling awesome-lint which readme you want to lint.

I think what you're getting at is that we're currently defaulting to `readme.md` (lowercase) if you don't pass a readme file on the command-line, whereas we should probably infer the name of the readme file using this [logic](https://github.com/sindresorhus/awesome-lint/blob/master/lib/find-readme-file.js).

**Comment by carlosperate** [comment]
Is the default filename changed in the latest release?
Using the same travis script, but installing version 0.2.0 instead of 0.3.0, doesn't give any error.
Perhaps I've never run `awesome-lint` correctly before by not adding a command line argument?

**Comment by transitive-bullshit** [comment]
A lot has changed since `v0.2.0`, but your use case should work going forwards.

We'll leave this issue open to track inferring the local readme file instead of defaulting to `readme.md`.

**Comment by carlosperate** [comment]
Sounds good, thank you Travis.

**Comment by styfle** [comment]
I have the same problem. I was using `v0.2.0` without any problems.
Then I started getting failures recently since the docs say to not pin the version `"awesome-lint": "*"`. https://travis-ci.org/styfle/awesome-online-ide/builds/417323728
